docs/examples/codecs/handling-data-types/Person.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use MongoDB\BSON\ObjectId;
+
+final class Person
```

Comment:
Noted that this differs from `handling-documents/Person.php` due to having an additional `createdAt` property.

I just want to confirm that none of the example files are exact duplicates. Feel free to resolve if that's correct.

Reply:
That is correct, they all have minor changes and thus can't be reused.